crowdsec-docs/docs/intro.mdx

```diff
@@ -26,8 +34,9 @@ The crowd-sourced aspect allows sharing attack information among users, enhancin
 In addition to the core "detect and react" mechanism, CrowdSec is committed to several other key aspects:
 
 - **Easy Installation**: Effortless out-of-the-box installation on all [supported platforms](/getting_started/versions_matrix.md).
-- **Simplified Daily Operations**: Use [cscli](/cscli/cscli.md) and the [hub](http://hub.crowdsec.net) for effortless maintenance and keeping your detection mechanisms up-to-date.
+- **Simplified Daily Operations**: Use the [console](http://app.crowdsec.net) and [cscli](/cscli/cscli.md) for effortless maintenance and keeping your detection mechanisms up-to-date.
```

Comment:
Maybe  precise the console is web and I guess cscli is explicit enough but we could give it a good adjective

- **Simplified Daily Operations**: Web UI administration via [CrowdSec's console](http://app.crowdsec.net) or the powerful Command line tool [cscli](/cscli/cscli.md) for effortless maintenance and keeping your detection mechanisms up-to-date.